util/util.go

```diff
@@ -593,6 +593,20 @@ func LowestNonZeroResult(i, j ctrl.Result) ctrl.Result {
 	}
 }
 
+// LowestNonZeroRetryAfterSeconds returns the lowest non-zero value of the two provided values.
+func LowestNonZeroRetryAfterSeconds(i, j int32) int32 {
```

Comment:
```suggestion
// LowestNonZero returns the lowest non-zero value of the two provided values.
func LowestNonZero(i, j int32) int32 {
```

Given that's just int32?

Reply:
Thanks for review 😄 
Yeah, right now the function is just used to compare the return value of `GetRetryAfterSeconds`, and the type of return value is int32.

And the suggestion on name change makes sense. Already updated